packages/amplify-graphql-relational-transformer/package.json

```diff
@@ -34,6 +34,7 @@
     "@aws-cdk/aws-appsync": "~1.124.0",
     "@aws-cdk/aws-dynamodb": "~1.124.0",
     "@aws-cdk/core": "~1.124.0",
+    "amplify-cli-core": "2.4.18",
```

Comment:
Latest is 2.5.1

Reply:
Also I think we shouldn't introduce this dependency. Probably we can introduce a types package specific to GraphQL transformers at a later time.